getting-started.md

```diff
@@ -153,6 +143,38 @@ The `logs` command lets you see textual output from your app, including `print`
 statements and unhandled exceptions. To avoid confusion from old log messages,
 you might want to use `flutter logs --clear` to clear the logs between runs.
 
+## Setting up your iOS device or simulator
+
+You must use a Mac, with Xcode installed, to develop iOS apps with Flutter.
+
+Install the following dependencies:
+
+- Install [Xcode 7](https://developer.apple.com/xcode/)
+- Install `ideviceinstaller`.
+  - `$ brew install ideviceinstaller`
+- Install the `ios-deploy` tool.
+  - `$ brew tap flutter/flutter`
+  - `$ brew install ios-deploy`
+
+
+## Adding iOS support to your project
+
+Inside of your Flutter app, run `flutter ios --init`
+to add iOS configuration files.
+
+You can shut down XCode, if it is started automatically.
+
+## Running your app in the iOS simulator
+
+- Run `Simulator.app`
+  - You can find Simulator.app in
+_Applications/Xcode/Contents/Developer/Applications_ or via _Spotlight_
+  - To ensure your simulator uses a device type of iPhone 5 or above,
```

Comment:
Specifically it's that we only support 64-bit devices at this time.  Once I have our new infra up and running we'll support both.